### DISCUSSION TITLE
mobile plugin is depracted in thruk 3.0 - whats the plan for mobile access in the future?

### DISCUSSION THREAD

**Discussion by ddraigddrud** [sni/thruk#1270] [General]
The mobile plugin works fantastically well with thruk <3.0, just updated to 3.0 and can see there is no longer a plugin available. The new site does sort of load in a mobile browser but there is so much info missing that it isn't useful anymore. Is there a new mobile plugin on the way, or is the plan to use the same interface for both desktop & mobile? ( running thruk 3.0 on omd 5.0)

**Comment by sni** [comment]
There are no plans to migrate the mobile plugin to Thruk 3. The idea is more like make all pages mobile friendly instead of duplicating some pages into a (outdated) mobile framework.

**Comment by ddraigddrud** [comment]
Thanks for the swift reply, that makes perfect sense - I'll have a look again at the latest version & try some different browsers

**Comment by ddraigddrud** [comment]
and it looks like the problem was......with our VPN
